docroot/modules/custom/va_gov_content_export/va_gov_content_export.routing.yml

```
@@ -0,0 +1,6 @@
+va_gov_content_export.content_export:
+  path: "/cms-export/{export_type}"
+  defaults:
+    _controller: 'Drupal\va_gov_content_export\Controller\ContentExport::redirectToFile'
+  requirements:
+    _access: 'TRUE'
```

Per @oksana-c's find, to allow anonymous access in maintenance mode.
```suggestion
    _access: 'TRUE'
    options:
      _maintenance_access: 'TRUE' 
```

---

We are not going to do this just yet without further testing.